### DISCUSSION TITLE
Applying a hunk (e.g. from a stash) should give some feedback

### DISCUSSION THREAD

**Discussion by doublep** [magit/magit#4982] [Ideas]
Problem: old stash no longer applies (too many conflicts, including in index). So I open it and try to apply hunk-by-hunk i.e. press `a`. I get zero feedback. If I accidentally, or thinking the key didn't work, press `a` again, I get error "patch doesn't apply" and it's even hard to tell if it really doesn't apply at all, or actually because I have applied it already.

There should be some positive feedback too ("...applied successfully"), otherwise nothing visible happens at all. On a side note, would be nice to move the point to the next hunk after having applied the current successfully.

**Comment by tarsius** [comment]
Makes sense.
